lib/screens/stop_watch.dart

```diff
@@ -0,0 +1,301 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/helpers/my_app1_icons.dart';
+
+class StopWatch extends StatefulWidget {
+  const StopWatch({super.key});
+
+  @override
+  State<StopWatch> createState() => _StopWatchState();
+}
+
+class _StopWatchState extends State<StopWatch> {
+  int hundredths = 0, seconds = 0, minutes = 0;
+  String digitHundredths = '00', digitSeconds = '00', digitMinutes = '00';
+  Timer? timer;
+  bool started = false;
+  List<String> laps = [];
+
+  void stop() {
+    timer!.cancel();
+    setState(() {
+      started = false;
+    });
+  }
+
+  void reset() {
+    timer!.cancel();
+    setState(() {
+      hundredths = 0;
+      seconds = 0;
+      minutes = 0;
+
+      digitHundredths = '00';
+      digitSeconds = '00';
+      digitMinutes = '00';
+
+      started = false;
+      laps.clear();
+    });
+  }
+
+  void addLaps() {
+    String lap = '$digitMinutes:$digitSeconds.$digitHundredths';
+
+    setState(() {
+      laps.add(lap);
+    });
+  }
+
+  void start() {
+    started = true;
+    timer = Timer.periodic(const Duration(milliseconds: 10), (timer) {
+      int localHundredths = hundredths + 1;
+      int localSeconds = seconds;
+      int localMinutes = minutes;
+
+      if (localHundredths > 99) {
+        if (localSeconds > 59) {
+          localMinutes++;
+          localSeconds = 0;
+        } else {
+          localSeconds++;
+          localHundredths = 0;
+        }
+      }
+      setState(() {
+        hundredths = localHundredths;
+        seconds = localSeconds;
+        minutes = localMinutes;
+        digitHundredths = (hundredths >= 10) ? '$hundredths' : '0$hundredths';
+        digitSeconds = (seconds >= 10) ? '$seconds' : '0$seconds';
+        digitMinutes = (minutes >= 10) ? '$minutes' : '0$minutes';
+      });
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints(maxHeight: 844, maxWidth: 388),
+      color: const Color(0xFF0E111C),
+      child: Column(
+        children: [
+          const Padding(
+            padding: EdgeInsets.fromLTRB(124, 58, 126, 0),
```

Comment:
This is not the way to center title, what if I run this on smaller phone? Left and Right paddings are too big.